### PR TITLE
Add drag-and-drop upload functionality to the top bar

### PR DESCRIPTION
**Pull request by whns**
Added a quick drag-and-drop upload feature (#9) to the topBar div and made the "turn your internet off" text in the instructions bold.

I borrowed heavily from [this tutorial on css-tricks](https://css-tricks.com/drag-and-drop-file-uploading/), but took out all of the jQuery usage.

### CODE REVIEW

File: scripts/css.css [hunk 1]
```diff
@@ -217,4 +217,8 @@ pre {
 
 .custom-file-open{
     cursor: pointer;
+}
+
+#topBar.is-dragover {
```
Comment (caelinsutch) [inline scripts/css.css:222]: I feel like a overlay would be good too

**Review by caelinsutch — CHANGES_REQUESTED** [review]
I would add some sort of indicator, maybe change the text to ` Click or drag image 📂` in the top  bar.

**Comment by caelinsutch** [comment]
Additional note, I added my own PR, #20, with a slightly less convulted solution. I couldn't get yours to work consistently in Chrome with MacOS.

**Comment by everestpipkin** [comment]
I merged in PR #20 which has duplicate functionality! Thank you again.

**Pull request closed without merging**